features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^the following dishes exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:dish, hash)
+  end
+end
+
+When(/^I am on the landing page$/) do
```

Comment:
We are changing this when landing page is merged. We are adding the dishes as a partial to the landing page. That's why it says landing